src/theme.js

```diff
@@ -38,6 +38,15 @@ export const theme = createMuiTheme({
       letterSpacing: 0.1,
       color: 'rgba(255, 255, 255, 0.6)',
     },
+    h6: {
```

Comment:
Is `headline6` and `h6` two different things, or maybe it's the same?
We already have some styles for `headline6` in [theme.js](https://github.com/DarthVanger/NeuralNotes/blob/master/src/theme.js#L16):
```
    headline6: {
      fontFamily: 'Roboto',
      fontStyle: 'normal',
      fontWeight: 500,
      fontSize: '19px',
      lineHeight: '22px',
    },
  ```